### ISSUE TITLE
Wording in docs about SecurityMiddleware might be confusing new users 

### ISSUE THREAD

**Issue by alexlouden** [evansd/whitenoise#122]
I came across (and answered) [this question](http://stackoverflow.com/questions/42261467/django-security-middleware-vs-whitenoise/) on stack overflow asking about why they can't use `SecurityMiddleware` and `WhiteNoiseMiddleware` together. They linked to [this section](http://whitenoise.evans.io/en/stable/django.html#enable-whitenoise) of the documentation.

The docs are currently:

> Edit your settings.py file and add WhiteNoise to the MIDDLEWARE_CLASSES list, above all other middleware apart from Django’s SecurityMiddleware:

```
MIDDLEWARE_CLASSES = [
  # 'django.middleware.security.SecurityMiddleware',
  'whitenoise.middleware.WhiteNoiseMiddleware',
  # ...
]
```

I think the commented out SecurityMiddleware and maybe the term "apart from" might have given them the wrong impression. May I suggest changing the wording and the commenting in the example?

> Edit your settings.py file and add WhiteNoise to the MIDDLEWARE_CLASSES list. The WhiteNoise middleware should be placed directly after the Django security middleware.

```
MIDDLEWARE_CLASSES = [
  'django.middleware.security.SecurityMiddleware',
  'whitenoise.middleware.WhiteNoiseMiddleware',
  # ...
]
```

**Comment by edmorley** [comment]
Ah I agree that wording is potentially confusing (my bad). 

Perhaps this is even clearer?
"The WhiteNoise middleware should be placed directly after the Django security middleware and before all other middleware."

**Comment by alexlouden** [comment]
Yeah that sounds good! I've made a PR.

**Comment by evansd** [comment]
Closed in #123
